src/Containers/VendorMarketSearch/VendorMarketSearch.scss

```diff
@@ -45,13 +46,19 @@
 
 .vendor-market-input {
   font-size: 1.5em;
-  border-radius: 20px;
+  // border-radius: 20px;
   outline: 0;
-  border: 0;
+  // border: 0;
   padding: 7px;
   text-align: center;
```

Comment:
The placeholder text is pretty difficult to see since it is a dark grey with the dark opaque background